sorting.js

```diff
@@ -0,0 +1,5 @@
+function sortingCards(cardsDeck) {
+  return cardsDeck.sort();
+
+  // I couldn't think of a way to implement in Javascript and the one I made isn't working
```

Comment:
Take a look at this https://www.geeksforgeeks.org/bubble-sort-algorithms-by-using-javascript/